group_vars/sn07.yml

```diff
@@ -279,7 +279,7 @@ condor_extra: |
 gie_proxy_dir: "{{ galaxy_root }}/gie-proxy/proxy"
 gie_proxy_git_version: main
 gie_proxy_setup_nodejs: nodeenv
-gie_proxy_virtualenv_command: /usr/bin/python3 -m virtualenv #"{{ pip_virtualenv_command }}"
+gie_proxy_virtualenv_command: /usr/bin/python3 -m venv #"{{ pip_virtualenv_command }}"
```

Comment:
For gie then the only change is switching from virtualenv to venv but it still uses system's Python (3.8) right?

Reply:
Yes, always use the absolute path, please.

Reply:
Most likely yes, the virtual environment is actually just [used to control Node.js environment](https://github.com/usegalaxy-eu/ansible-gie-proxy/blob/ca6b025bc06ae8c0de98c339b1c39e1709e86304/tasks/nodejs.yml#L8C1-L27) using [nodeenv](https://pypi.org/project/nodeenv). The proxy itself is written in JavaScript.